woocommerce/woocommerce-android
# Feature/9963 blaze campaign list

Part of: #9963


### Description

Adds new screen to display a list with all the Blaze campaigns

### Testing instructions

1. Launch the app and click on "Show all" button in My Store tab Blaze campaign card. 
2. Check the list of hard-coded test campaigns is displayed matching the Figma design: mBhyoV7OPhFnOugxLy6SMS-fi-627%3A10640

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/faa97187-de13-4fd5-8c9c-3ae864c24471
